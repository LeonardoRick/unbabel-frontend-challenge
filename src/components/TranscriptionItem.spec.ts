import TranscriptionItemVue from '@/components/TranscriptionItem.vue';
import { mount } from '@vue/test-utils';

describe('TranscriptionItem', () => {
    it('should mount', () => {
        expect(
            mount(TranscriptionItemVue, { props: { item: { id: 1, voice: 'string', text: 'string' } } }).html()
        ).toMatchSnapshot();
    });
});
