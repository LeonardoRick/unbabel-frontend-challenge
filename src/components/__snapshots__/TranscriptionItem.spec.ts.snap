// Vitest Snapshot v1

exports[`TranscriptionItem > should mount 1`] = `
"<li><input type=\\"checkbox\\" class=\\"item-checkbox\\"><img src=\\"/src/assets/images/person.svg\\" alt=\\"user image\\">
  <div class=\\"text-wrapper\\"><input type=\\"text\\" class=\\"voice\\"><textarea></textarea></div><button type=\\"button\\" class=\\"reset-button-style delete-button\\"><img src=\\"/src/assets/images/delete.svg\\" alt=\\"delete transcription\\"></button>
</li>"
`;
